docs/integration/ldap.md

```diff
@@ -0,0 +1,141 @@
+You don't need to change it, just confirm this is the setting you have in place.
+
+```bash
+PULP_GALAXY_AUTHENTICATION_CLASSES=['rest_framework.authentication.SessionAuthentication','rest_framework.authentication.TokenAuthentication','rest_framework.authentication.BasicAuthentication']
+PULP_GALAXY_DEPLOYMENT_MODE=standalone
+```
+
+Pulp container requires this to be set in order to provide docker registry compatible token authentication.
+https://docs.pulpproject.org/pulp_container/authentication.html
+
+```bash
+PULP_TOKEN_AUTH_DISABLED=true
+```
+
+`django_auth_ldap` must be included as the first authentication backend.
+
+```bash
+PULP_AUTHENTICATION_BACKENDS=["django_auth_ldap.backend.LDAPBackend","django.contrib.auth.backends.ModelBackend","guardian.backends.ObjectPermissionBackend","pulpcore.backends.ObjectRolePermissionBackend"]
+```
+
+Specific django_auth_ldap settings
+
+!!! tip
+    depending on the LDAP server some of the following settings might need change.
+
+The following keys are **required** in order to have LDAP enabled:
+
+```bash
+PULP_AUTH_LDAP_SERVER_URI="ldap://ldap:10389"
+PULP_AUTH_LDAP_BIND_DN="cn=admin,dc=planetexpress,dc=com"
+PULP_AUTH_LDAP_BIND_PASSWORD="GoodNewsEveryone"
+PULP_AUTH_LDAP_USER_SEARCH_BASE_DN="ou=people,dc=planetexpress,dc=com"
+PULP_AUTH_LDAP_USER_SEARCH_SCOPE="SUBTREE"
+PULP_AUTH_LDAP_USER_SEARCH_FILTER="(uid=%(user)s)"
+PULP_AUTH_LDAP_GROUP_SEARCH_BASE_DN="ou=people,dc=planetexpress,dc=com"
+PULP_AUTH_LDAP_GROUP_SEARCH_SCOPE="SUBTREE"
+PULP_AUTH_LDAP_GROUP_SEARCH_FILTER = "(objectClass=Group)"
+PULP_AUTH_LDAP_GROUP_TYPE_CLASS="django_auth_ldap.config:GroupOfNamesType"
+```
+
+Optional variables:
+
+```bash
+PULP_AUTH_LDAP_USER_ATTR_MAP={first_name="givenName", last_name="sn", email="mail"}
+# NOTE: the above is formatted as a toml hashmap
+
+PULP_AUTH_LDAP_MIRROR_GROUPS=true
+# The above is what enabled group mirroring
+```
+
+You can limit which groups are mirrored if you don't want all the groups to be added do Hub.
+
+```bash
+AUTH_LDAP_MIRROR_GROUPS_EXCEPT=['foobar']
+# this syncs all groups except the `foobar`
+```
+
+Require a specific group for all users
+
+```bash
+AUTH_LDAP_REQUIRE_GROUP='hub_users'
+# Only users belonging to this group will be allowed to login
+```
+
+TLS verification
+
+```bash
+# Make ldap to call start_tls on connections
+PULP_AUTH_LDAP_START_TLS=true
+
+# If using self signed certificates set this
+PULP_GALAXY_LDAP_SELF_SIGNED_CERT=true
+```
+
+Logging:
+
+```bash
+# Enable LDAP logging handler
+PULP_GALAXY_LDAP_LOGGING=true
+```
+
+Cache
+
+```bash
+# Change the caching lifetime (for groups and users search)
+PULP_AUTH_LDAP_CACHE_TIMEOUT=3600
+```
+
+
+More settings can be found on https://django-auth-ldap.readthedocs.io/en/latest/reference.html#settings
+
+
+!!! warning
+    If you need to change the values of `AUTH_LDAP_GROUP_SEARCH` and `AUTH_LDAP_USER_SEARCH` which uses the binding to the class `LDAPSearch`
```

Comment:
Can these be set up to not require changes in `dynaconf_hooks.py`?

Reply:
This is `/etc/pulp/dynaconf_hooks.py` and not the `dynaconf_hooks.py` we have inside our app folder.